learningequality/kolibri-design-system
# Add copyToClipboard, home, (un)publishedResource icons, generate RST replacements

## Description

Currently, there is no `copy` icon with the `Outline` style in KDS - which is necessary in Studio.

In Studio, however, we have not fully applied `KIcon` and `KIconButton` and so forth - so there are many icons used there by way of Vuetify's handling of material icons.

But - @radinamatic needs these icons on hand so she can update documentation with access to all of the icons we see in our products, even if the product isn't currently getting the icon from KDS.

Thus I've added it here.

#### Issue addressed


Fixes #141 

Also - the `unpublishedChanges` and `publishedResource` icons are `tokens.success`.

Initially, the closest color we had was `brand.secondary` which is also `tokens.focusOutline` - this matched the similar version from Vuetify currently used in Studio.

That did not pass muster as an icon for a11y purposes - so I've opted to use that darker green `tokens.success` color which passes in the icon context:

![changes-publish-icon-color-contrast-using-success](https://user-images.githubusercontent.com/6356129/116613638-c58d0100-a8ed-11eb-8e90-d31c1665d21d.png)


## Steps to test

1. Go to the deployed Netlify testing site at the bottom here in the "Github Actions" section.
2. Go to icons
3. Type in "copy" to see the two copy icons.
4. Scroll to the bottom for the RST Replacement text - see that the `copyToClipboard` icon is there and is acceptable.

@jtamiace - do you agree that `copyToClipboard` is a solid alias for that icon? It matches the actual use of it in Studio anyway. See @indirectlylit 's comment from #141 

![screenshot-github com-2021 04 09-16_23_29](https://user-images.githubusercontent.com/6356129/114249631-05e10b00-9950-11eb-9a52-2902dfe27267.png)

---

@radinamatic does this solve the concerns you had in #141?
